nunit/MVP-Advance/SeleniumNUnit/Tests/Tests.cs

```diff
@@ -85,7 +85,40 @@ public void TC3c_WhenIAddInvalidDataThenIAssert()
             AssertInvalidData(10, 11, 12, "NegativeTC"); //need test data
         }
 
- 
+        [Test]
+        public void InvalidLang_1()
+        {
+            test = extent.CreateTest(TestContext.CurrentContext.Test.Name);
+            profileObj = new Profile();
+            profileObj.InvalidLanguage_1(2, "LangNegTest");
+            AssertInvalidLang(4, "LangNegTest");
+        }
+
+        [Test]
+        public void InvalidLang_2()
```

Comment:
the above comment applies here InvalidLang_1 vs InvalidLang_2 ?

Reply:
I have done name changes